script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt

```diff
@@ -139,7 +139,8 @@ But, unbelievable as it is, the thing in front of me now is definitely real.
 [sha:44fe03efbe94d42619e47b605eb1a0214ccd2202]{
 -- Page 161, Offset 21748.
 -- 槙久の部屋の窓は、一面の壁だった。
-The window in Makihisa's room has been completely walled off.
+//Walled off refers more closely to literal walls 
+The window in Makihisa's room has been completely sealed shut.
```

Comment:
I mean the CG is pretty heavy duty, I might lean towards keeping it walled.

Reply:
idk, I think I agree with this edit. I can't really explain it other than "walled off" makes me think of the level of Ghost Master where the old lady has sealed a welder behind literal bricks.

The CG _is_ more sealed off though for a more coherent explanation

Reply:
For the love of god, Montresor.